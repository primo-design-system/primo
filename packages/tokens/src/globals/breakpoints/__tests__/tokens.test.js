const breakpointTokens = require('../tokens.json');

describe('breakpoint tokens', () => {
  test(': matches snapshot', () => {
    expect(breakpointTokens).toMatchSnapshot();
  });
});
