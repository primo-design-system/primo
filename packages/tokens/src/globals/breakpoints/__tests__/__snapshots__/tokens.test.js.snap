// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`breakpoint tokens : matches snapshot 1`] = `
Object {
  "breakpoints": Object {
    "lg": Object {
      "value": "1024px",
    },
    "md": Object {
      "value": "768px",
    },
    "sm": Object {
      "value": "375px",
    },
    "xl": Object {
      "value": "1280px",
    },
    "xs": Object {
      "value": "320px",
    },
  },
}
`;
